examples/Basic/console.py

```diff
@@ -81,11 +81,29 @@ def queue(self):
         self.manager.queue(experiment)
 
 
+class MainConsole(ManagedConsole):
+
+    def __init__(self):
+        super().__init__(
+            procedure_class=TestProcedure,
+        )
+
+        self.queue()
+
+    def queue(self):
+        filename = self.get_filename(self.directory)
+        procedure = TestProcedure()
+        results = Results(procedure, filename)
+        experiment = self.new_experiment(results)
+
+        self.manager.queue(experiment)
+
+
```

Comment:
I am not sure, I catch the meaning of this class. I think `ManagedConsole` does not support queuing as there is no runtime user interface (just command line options). I will keep digging in the code.

Reply:
So this brings `ManagedConsole` more inline with the code pattern of `ManagedWindow` so it can use the `Manager` class. At the moment `queue()` is just like `start()`, but if `ManagedConsole` will support procedure sequencer it will need to have a queue mechanism. That way you can pass a sequence file to `ManagedConsole` and a bunch of experiments will get queued, like `ManagedWindow` does now.

Reply:
Got it. I have done some restructuring in my commit dd20ff69d355b395a8dc439e728a9dae7ecdcaa9. Could You please have a look ?

Reply:
Yea looks great, I pulled it down and everything is running as expected.